### PR TITLE
Add Custom Crosshairs

### PR DESCRIPTION
**Pull request by pythoncat1**
## Description

I have added the ability to use svgs from `client/public/img/crosshairs` as custom crosshairs. You can select a crosshair through the customize menu, like how you would for skins or emotes.
This feature is **NOT COMPLETE** as these features are still missing:
- Not showing pointer cursor at same time while going on game-ui buttons (like heals or slots)
- Changing cursor size/enabling custom cursor in settings
- More cursors (currently have two)

## Type of change

* **feat**: A new feature.

## How Has This Been Tested?

This has been hosted locally and played with.

### CODE REVIEW

**Comment by leia-uwu** [comment]
image
uhhh what

**Comment by pythoncat1** [comment]
i forgot to gitignore .history sorry

**Comment by ei-pi** [comment]
was gonna say to mark incomplete features as drafts, but Leia beat me to it haha

**Comment by pythoncat1** [comment]
@leia-uwu i can close this PR and make another without .history if you would like

**Comment by leia-uwu** [comment]
just remove .history, rebase and squash the commits and force push

**Comment by pythoncat1** [comment]
how can I do those through git web

**Comment by leia-uwu** [comment]
uhhh no idea i don't use git web

**Comment by leia-uwu** [comment]
k i removed and rebased to yeet the .history stuff

File: common/src/definitions/crosshairs.ts
Comment (leia-uwu) [inline common/src/definitions/crosshairs.ts:1]: this should probably be somewhere in the client and not /common since crosshairs are client side only

File: client/src/scripts/ui.ts
Comment (leia-uwu) [inline client/src/scripts/ui.ts:1]: why so many unrelated code style changes

**Review by leia-uwu — CHANGES_REQUESTED** [review]

File: client/src/scripts/ui.ts
Comment (pythoncat1) [reply client/src/scripts/ui.ts:1]: VSC auto formats my code idk

File: common/src/definitions/crosshairs.ts
Comment (pythoncat1) [reply common/src/definitions/crosshairs.ts:1]: yeah maybe in utils

**Comment by pythoncat1** [comment]
I have rewritten this code and will make a new pr soon.
